src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java

```diff
@@ -63,11 +63,11 @@ public class AribaPluginConfig extends ReferencePluginConfig {
   private final String systemType;
 
   @Macro
-  @Description("Realm name for the customer.")
+  @Description("Target Realm name from which data is to be extracted.")
```

Comment:
Don't think you need Target here. How about "Realm name from which data is to be extracted."

Reply:
Added